pyrightconfig.json

```diff
@@ -1,9 +1,12 @@
 {
     "include": [
       "./elements/pl-hidden-hints",
-      "./elements/pl-order-blocks",
```

Comment:
Removed since functions without annotations are now ignored (and without them, the typechecking does almost nothing anyway).